common/utils.js

```diff
@@ -8,3 +8,19 @@ export function zeroPad(i) {
   }
   return i;
 }
+
+/**
+* Convert Celsius to Fahrenheit
+* @param {object} data - WeatherData -
+*/
+export function tempCFConv(data) {
```

Comment:
rename `tempCFConv(data) `to: `toFahrenheit(data)`

Reply:
fixed